containers/Makefile

```diff
@@ -174,15 +211,17 @@ build/config-with-versions.yml: versions.yml config.yml ../tools/merge-yaml.py
 
 
 clean:
-	rm -rf build/
-
-
-.PHONY:                                 \
-    all                                 \
-    nginx-proxy   nginx-proxy-clean     \
-    nginx-storage nginx-storage-clean   \
-    postgresql    postgresql-clean      \
-    concent-api   concent-api-clean     \
-    verifier      verifier-clean        \
-    docker-clean                        \
+	rm -rf build/ dist/
```

Comment:
`clean` target is usually meant to remove only intermediate files.

There should be a separate `distclean` for removing `dist/`. See [GNU Coding Standards > Standard Targets for Users](https://www.gnu.org/prep/standards/html_node/Standard-Targets.html).

Reply:
Done.